components/strava/sources/activity-created/activity-created.js

```diff
@@ -4,13 +4,16 @@ module.exports = {
   key: "strava-activity-created",
   name: "Activity Created",
   description: "Emits an event when a new activity is created",
```

Comment:
Did you setup ESLint locally in your editor? I like to do that so I catch these errors there.

Reply:
It has some warnings, I am correcting them